swagger.yaml

```diff
@@ -1459,15 +1459,25 @@ components:
             type: string
             format: uuid
         roles:
-          description: User role
+          description: User roles, can be empty or filled if includeRoles pameter is used. Defines the access level of the user.
           readOnly: true
           type: array
           items:
             $ref: '#/components/schemas/UserRole'
+        jobTitle:
```

Comment:
I get why jobTitle is a good option, but I also find ```position``` a great fit :D

Reply:
You're right, let's go with Position so that we don't add the 3rd confusing term with jobTitle :D 
What about the "roles" property, is it still needed?